packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -527,4 +527,18 @@ aar artifact already exists, continuing...''',
 
     await promotePatch(patchId: patch.id, channel: channel);
   }
+
+  Future<List<AppUsage>> getUsage() async {
+    final progress = logger.progress('Fetching usage');
+    final List<AppUsage>? appsUsage;
+    try {
+      appsUsage = await codePushClient.getUsage();
+      progress.complete();
+    } catch (error) {
+      progress.fail(error.toString());
+      exit(ExitCode.software.code);
+    }
+
+    return appsUsage;
```

Comment:
nit: this could just return from within the try and you wouldn't need to declare it separately on line 533